lib/YuiRestClient.pm

```diff
@@ -42,6 +45,16 @@ sub get_host {
     return $host;
 }
 
+sub get_logger {
+    unless (defined $logger) {
```

Comment:
YuiClient::Logger->get_instance()->debug();

Reply:
Moved the initialization to main.pm, so now the logger can be used as `YuiRestClient::Logger->get_instance()->debug`, or even `YuiRestClient::Logger->debug`, because we are sure that the logger is initialized already in very beginning of Job Run (in main.pm)